lib/selectivity/rspec.rb

```diff
@@ -44,25 +44,25 @@ def find_selectivity_input(from, options)
         find(:div, "##{label[:for]}", options)
       end
 
-      def multiselect?(input)
+      def _selectivity_multiselect?(input)
         input.first('.selectivity-multiple-input-container').present?
       end
 
-      def select!(input, item)
+      def _selectivity_select!(input, item)
         input.click
 
         within 'div.selectivity-dropdown' do
-          if (find(:xpath, "//div[@class='selectivity-result-item'][contains(text(), '#{item}')]")).visible?
+          if find('div.selectivity-result-item', text: item).visible?
```

Comment:
Alright, thanks for details! I think it has already been reported - https://github.com/teampoltergeist/poltergeist/issues/317.

I admit I didn't check on Poltergeist though. Will merge this now, play with the driver and try to find out a sufficient solution that uses `xpath`. You are also very welcome to try yourself :wink: